var/spack/repos/builtin/packages/usearch/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
+
+
+class Usearch(Package):
+    """USEARCH is a unique sequence analysis tool with thousands of users
+       world-wide.
+
+       Note: A manual download is required for USEARCH.
+       Spack will search your current directory for the download file.
+       Alternatively, add this file to a mirror so that Spack can find it.
+       For instructions on how to set up a mirror, see
+       http://spack.readthedocs.io/en/latest/mirrors.html"""
+
+    homepage = "http://www.drive5.com/usearch/"
+
+    version('10.0.240', '05192b6d5e291530c190a19a3cc82b53', expand=False)
+
+    def url_for_version(self, version):
+        return "file://{0}/usearch{1}_i86linux32".format(os.getcwd(), version)
+
+    def install(self, spec, prefix):
+        mkdirp(prefix.bin)
+        install('usearch10.0.240_i86linux32', prefix.bin.usearch)
```

Comment:
```python
install('usearch{0}_i86linux32'.format(self.version), prefix.bin.usearch)
```
Not that I think we'll be adding new version frequently, but it's always good to plan ahead!